### PR TITLE
Feat: product listing added

### PR DESCRIPTION
**Pull request by ShoaibTamboli**
product listing added

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
In product listing page remove overflow from x-axis

File: pages/products/products.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ display: flex;
+}
+.product-listing{
+    background-color: yellow;
+    height: 200px;
+}
+
+.radio-label{
+    display: inline !important ;
+}
```
Comment (rishichourasia) [inline pages/products/products.css:24]: Aviod using Important keyword , check which code is overwriting your styles

**Review by rishichourasia — COMMENTED** [review]
Few Suggestions - 
```
1 - Avoid using important keyword , use inspect element  to  see which styles is getting applied. 
2 - Use rem as units and not px . 
3 - Use generic height styling like auto or fit-content ( Issues in responsiveness) 
4 Navbar CTA ( wish list and cart ) are in the middle , use margin left auto to shift them to right.

Small changes but will improve ui prespectives

File: pages/products/products.css (hunk 1 above)
Comment (ShoaibTamboli) [reply pages/products/products.css:24]: Actually, one element from CL is overriding, will fix this.

**Pull request merged**